eslint.config.mjs

```diff
@@ -22,6 +22,7 @@ export default typescriptEslint.config(
       "@typescript-eslint/no-explicit-any": 0,
       "@typescript-eslint/no-non-null-assertion": 0,
       "@typescript-eslint/no-var-requires": 0,
+      "prettier/prettier": ["error", { "endOfLine": "auto" }] // match .gitattributes
```

Comment:
Can we change gitattribute though? The unit test currently is manually normalizing line end and we should just prefer LF everywhere.